zookeeper-server/src/main/java/org/apache/zookeeper/server/persistence/TxnLogToolkit.java

```diff
@@ -332,7 +338,15 @@ private static String getDataStrFromTxn(Record txn) {
 
         return txnData.toString();
     }
-    
+
+    private static String checkNullToEmpty(byte[] data) {
```

Comment:
Do we have any utility class for this stuff?

Reply:
Since the parameters is `byte[] data`, I think we may don't hava any one.